src/transaction/components/TransactionButton.tsx

```diff
@@ -26,6 +25,8 @@ export function TransactionButton({
     transactionId,
   } = useTransactionContext();
 
+  const { address } = useAccount();
```

Comment:
Should `useAccount` be added even in small components. Or should be synced at Provider level?

Open question

Reply:
let's go ahead with your intuition.

Reply:
sounds good